controllers/user/login.js

```diff
@@ -35,13 +37,11 @@ router
     .get(renderForm)
     .post((req, res, next) => {
         // @TODO consider rate limiting?
-        passport.authenticate('local', (err, user, info) => {
-
+        passport.authenticate('local', (err, user) => {
             if (!user) {
                 // User not valid, send them to login again
-                res.locals.errors = [{ msg: info.message }];
-                res.locals.formValues = req.body;
-                return renderForm(req, res);
+                const errors = [{ msg: 'Username or password is invalid' }];
```

Comment:
Noticed whilst testing this that the errors returned to the front-end was the raw passport error. For login we only want to render a generic error that the log in credentials weren't valid. There was no express-validator code on this route before, we may want to add some basic validation like the forgotten password screen, if only to check that the username is an email address.